share/initialization/common

```diff
@@ -90,6 +90,8 @@ fi
 
 # some useful shortcuts
 alias -- -='cd -'
+alias h='history 25'
+alias j='jobs -l'
```

Comment:
The `jobs` builtin already prints out existing jobs, should I remove the `-l` flag or remove this alias entirely?